include/z64math.h

```diff
@@ -95,7 +95,7 @@ typedef struct {
 /**
  * Trig macros
 */
-#define DEGF_TO_BINANG(degreesf) (s16)(degreesf * 182.04167f + 0.5f)
+#define DEGF_TO_BINANG(degreesf) (s16)(degreesf * 182.04167f + .5f)
```

Comment:
im worried that this specific fix is only relevant for camera. if it starts breaking stuff outside of camera then it should probably be a local macro

Reply:
I believe Krim made it for camera. I don't know if anything else even uses that.

Reply:
yea he did, but it was put in z64math with the intention of being used everywhere i guess. just something to keep in mind

Reply:
speaking of macros in math, BINANG_SUB should probably be SUB16